_posts/2023-11-14-specify-file-audit-log-encoding.adoc

```diff
@@ -0,0 +1,9 @@
+---
+layout: post
+title: 'Change the default encoding of the audit log file in WildFly 29'
+date: 2023-11-14
```

Comment:
While you're making an update, would also be good to update the date here and in the filename to today's date. Thanks @ivassile!